### PR TITLE
snaking_static effect

### PR DESCRIPTION
**Pull request by savaughn**
ain't much, but better than nothin :rofl:

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.03%) on `us`.

File: src/effects/snaking_static.c [hunk 1]
```diff
@@ -3,12 +3,142 @@
...
+            }
+        }
+        data->unk_24 = (shim_sin_deg(((data->lifeTime * 180) / data->timeLeft)) * 1024.0f);
+        if (data->unk_24 >= 256) {
+            data->unk_24 = 255;
+        }
+        data->lifeTime++;
+    }
+    data = effect->data.snakingStatic;
+    data++;
```
Comment (nanaian) [inline src/effects/snaking_static.c:137]: Can't do + 1?

Comment (savaughn) [reply src/effects/snaking_static.c:137]: do you mean like this?
```
data = effect->data.snakingStatic + 1;
```

Comment (ethteck) [reply src/effects/snaking_static.c:137]: the way you have it is better imo, since it matches the way we normally use data and handle it in these loops

**Pull request merged**